client/src/components/core/ms-modal/MsAlertModal.vue

```diff
@@ -15,9 +15,18 @@
   />
 </template>
 
+<script lang="ts">
```

Comment:
Probably a rebase gone wrong, `MsAlertModalConfig` should be imported, not defined here.

Reply:
Done